helpdesk/management/commands/get_email.py

```diff
@@ -39,20 +43,34 @@
 from helpdesk.lib import send_templated_mail, safe_template_context
 from helpdesk.models import Queue, Ticket, FollowUp, Attachment, IgnoreEmail
 
+import logging
+from time import ctime
+
 
 class Command(BaseCommand):
     def __init__(self):
         BaseCommand.__init__(self)
 
-        self.option_list += (
+        # Django 1.7 uses different way to specify options than 1.8+
+        if VERSION < (1, 8):
+            self.option_list += (
             make_option(
                 '--quiet', '-q',
                 default=False,
                 action='store_true',
                 help='Hide details about each queue/message as they are processed'),
             )
 
-    help = 'Process Jutda Helpdesk queues and process e-mails via POP3/IMAP as required, feeding them into the helpdesk.'
+    help = 'Process django-helpdesk queues and process e-mails via POP3/IMAP or from a local mailbox directory as required, feeding them into the helpdesk.'
+
+    def add_arguments(self, parser):
```

Comment:
So, I assume correctly that this would have failed in Django > 1.7, since we did not provide the ``add_argument``?
Maybe we could add some tests that cover this?